development-defaults.ini

```diff
@@ -0,0 +1,6 @@
+[enums]
+[[indie_genre]]
+genre1 = 'Genre 1'
+genre2 = 'Genre 2'
+genre3 = 'Genre 2'
+genre4 = 'Genre 2'
```

Comment:
I wasn't sure what genres should exist, so these are obviously just placeholders.  Changing what genres show up on the application form is just a simple config change.